app.py

```
@@ -51,7 +51,11 @@
 
 # 初始化 Flask 应用
 load_dotenv()
-app = Flask(__name__)
+# 前端构建产物落在 ./admin-ui（见 frontend/vite.config.ts 的 outDir），
```

Ruff (0.15.18)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)